File: lang_speed_test/load.py
import time

def load(N):
	x = []
	i = 0
	s = time.time()
	while i<N:
		x.append("document_%07d" %i)
		i+=1
	e = time.time()

	print e-s


load(10000000)


	
		
	

File: lang_speed_test/load.cc
#include <iostream>
#include <sstream>
#include <vector>
#include <iomanip>
using namespace std;

int main() {
    int N = 10000000;
    vector<string> items(N);
    int i;
    time_t start, end;

    start = time(NULL);
    for (i=0; i<N; i++) {
        stringstream ss;
        ss<<"document_"<<setfill('0')<<setw(7)<<i;
        items[i] = ss.str();
    }
    end = time(NULL);
    cout<<"Time taken:"<<end-start<<endl;

}
